response/main.tex

```diff
@@ -0,0 +1,144 @@
+\documentclass{article}
+\usepackage[margin=2cm, includefoot, footskip=30pt]{geometry}
+\setlength\parindent{0pt}
+\setlength{\parskip}{1em}
+\usepackage{authblk}
+ 
+\title{Stability of defection, optimisation of strategies and the limits of
+memory in the Prisoner's Dilemma RESPONSE TO REVIEWS}
+
+\author{Nikoleta E. Glynatsi, Vincent A. Knight}
+ 
+\begin{document}
+
+\maketitle
+
+We would like to open this response by thanking the reviewers for their
+thoughtful comments and suggestions. We have fully taken their comments on board
+and made the necessary changes.
+
+The main modifications include the improvement of the theorems' proofs and the
+generalisation of our results regardless the PD payoff matrix. Moreover, we
+included more details of our methodology, such as the mathematical explanation
+of the SSE method, and a discussion based on the results of Hilbe et al from
+``Partners and rivals in direct reciprocity'' (Nature Human Behaviour, 2018).
+Finally, following the suggestions of Reviewer 2 we improved the narrative of
+the manuscript which involved changing the title of the manuscript to 
+``Best response memory-one strategies with a theory of mind: the limitations
+of extortion and restricted memory''.
+
+We hope that our manuscript will be considered for publication following our
+modifications.
+
+\section{Response}
+
+\textbf{Regarding the comments of Reviewer 1:}
+
+\begin{verbatim}
+     ``Theorem 2: First, it is not clear how equation 17 is obtained.''
+\end{verbatim}
+
+We have added further details on how to obtain the equation in the proof of Theorem 2
```

Comment:
Can we for each of these, add a bit more detail.

So for example, here we'd say:

```
This comment point out that we had skipped the step of multiplying the steady state vector by the utilities \(v\dot (RPTS)\). This has been added in the proof of what used to be T 2 but is now T1.
```